src/app/helptext/system/ca.ts

```diff
@@ -20,7 +20,17 @@ export const helptext_system_ca = {
     },
 
     create_type: {
-      placeholder: T("Type")
+      placeholder: T("Type"),
+      tooltip: T('Choose between <i>Internal CA</i>, <i>Intermediate CA</i>, and \
+ <i>Import CA</i>. An <i>Internal CA</i> functions like a publicly trusted CA \
+ to sign certificates for an internal network. They are not trusted outside \
+ the private network. An <i>Intermediate CA</i> lives between the root and end \
+ entity certificates and their main purpose is to define and authorize the \
```

Comment:
Singular/Plural agreement error
```suggestion
 entity certificates and its main purpose is to define and authorize the \
```